spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1109,6 +1111,106 @@ The critical differences between Offset and Keyset pagination lie in their retri
 - Offset pagination uses a fixed page size and retrieves data based on page number and size.
 - Keyset pagination relies on a unique key or unique combination of values (the keyset) for an entity relative to which it determines the next page or previous page.
 
+==== Offset Pagination in Jakarta Data
+
+Offset pagination is a popular method for managing and retrieving large datasets efficiently. It is based on dividing the dataset into pages containing a specified number of elements. This method allows developers to retrieve a subset of the dataset by identifying the page number and the maximum number of elements per page.
+
+Offset pagination is motivated by the need to provide efficient navigation through large datasets. Loading an entire dataset into memory at once can be resource-intensive and lead to performance issues. By breaking the dataset into smaller, manageable pages, offset pagination improves performance, reduces resource consumption, and enhances the overall user experience.
+
+Offset pagination offers several key features that make it a valuable approach for managing and retrieving large datasets in a controlled and efficient manner:
+
+- *Page Size*: The maximum number of elements to be included in each page is known as the page size. This parameter determines the subset of data retrieved with each pagination request.
+
+- *Page Number*: The page number indicates which subset of the dataset to retrieve. It typically starts from 1, representing the first page, and increments with each subsequent page.
+
+- *Efficient Navigation*: Offset pagination allows efficient dataset navigation. By specifying the desired page and page size, developers can control the data retrieved, optimizing memory usage and processing time.
+
+- *Sequential Order*: Elements are retrieved sequentially based on a predefined criterion, such as ascending or descending order of a specific attribute, like an ID.
```

Comment:
The plural form should be used here because the pattern allows ordering by multiple.
```suggestion
- *Sequential Order*: Elements are retrieved sequentially based on predefined criteria, such as ascending or descending order of a specific attribute, like an ID.
```